sdk/purview/purview-datamap-rest/src/purviewDataMapClient.ts

```diff
@@ -37,8 +39,14 @@ export default function createClient(
       scopes: options.credentials?.scopes ?? ["https://purview.azure.net/.default"],
     },
   };
+  const client = getClient(endpointUrl, credentials, options) as PurviewDataMapClient;
 
-  const client = getClient(baseUrl, credentials, options) as PurviewDataMapClient;
+  client.pipeline.removePolicy({ name: "ApiVersionPolicy" });
```

Comment:
this is the case where we shouldn't have client level apiVersion because not all operations have this parameter.